test/built-ins/GeneratorPrototype/return/try-finally-set-property-within-try.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2022 Bo Pang. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 25.3.1.3
```

Comment:
`es6id` is deprecated in favor of `esid`, see https://github.com/tc39/test262/blob/main/CONTRIBUTING.md#frontmatter . Also note that 25.3.1.3 doesn't even exist in the current edition of ECMAScript, but if it did exist then it would be a subsection of [Abstract Operations For DataView Objects](https://tc39.es/ecma262/multipage/structured-data.html#sec-abstract-operations-for-dataview-objects), which doesn't correspond with the content of this test.
```suggestion
esid: sec-generator.prototype.return
```